stock_picking_date_deadline_syncs_scheduled_date/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+{
+    "name": "Stock Picking Date Deadline syncs Scheduled Date",
+    "summary": "Sync Scheduled Date with Date Deadline in Stock Picking",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Inventory/Inventory",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["Shide"],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["sale_stock"],
```

Comment:
With the behaviour of this module, scheduled date is not only updated for sale moves, but also for purchases or productions .
With this module you want to change all the scheduled date behaviour for stock moves? If so, you would have to remove the dependency with sale_stock and put stock instead.
Do you only want to have this behaviour for sales? Then you would have to add some kind of restriction, e.g. only update date if the move has a sales order line.

Reply:
No, the behavior is generic, but Sales are the only module that uses date deadline (don't know if MRP does too... but for sure if comes from a sale).

So, the only real case is when Deadline is set, and for so, Sales must be installed.

In the early development process, I depends only from stock, because the field is on stock. But later when using the module, no purchase nor manual picking creations wrote Deadline fields. This is why I've chosen sale_stock.